api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/ClaimDTO.java

```diff
@@ -70,4 +70,8 @@
 
     @JsonProperty("claimant")
     private ClaimantDTO claimant;
+
+    @ApiModelProperty(notes = "Eligibility override decision based on eligibility outcome and until date")
```

Comment:
It also contains children dobs.

Reply:
Suggest `An override of the DWP/HMRC eligibility decision`